src/main/java/pl/mateusz/zadanie20/UserRepository.java

```diff
@@ -0,0 +1,26 @@
+package pl.mateusz.zadanie20;
+
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public class UserRepository {
+
+    private List<User> userList;
+
+    public UserRepository(List<User> userList) {
+        this.userList = userList;
```

Comment:
Ten konstruktor powinien być bez żadnego parametru. Jeżeli jako parametr dodałeś listę, to wskazujesz, że tę listę powinien utworzyć Spring i ją tutaj wstrzyknąć, ale w tym przypadku to Ty powinieneś ją utworzyć:
```
private List<User> userList = new ArrayList<>();
```